airflow-core/src/airflow/ui/src/pages/TaskInstance/Logs/Logs.tsx

```diff
@@ -68,6 +69,9 @@ export const Logs = () => {
   const toggleWrap = () => setWrap(!wrap);
   const toggleFullscreen = () => setFullscreen(!fullscreen);
 
+  useHotkeys("w", toggleWrap);
```

Comment:
We have wrap on the dag code and task log previews. Do we want to have this hotkey adjust those too?

Reply:
Good idea. I have added it in 8339a38309422812e0e2cf7e2c470a62f4345c3e . The code tab part might be of less usage but still something users can use as needed. Thanks.